erpnext/public/js/controllers/transaction.js

```diff
@@ -183,6 +183,9 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 				() => set_value('price_list_currency', currency),
 				() => set_value('status', 'Draft'),
 				() => set_value('is_subcontracted', 'No'),
+				() => {
```

Comment:
Changes should not be made in the transaction controller just for one specific use case. This might trigger unwanted changes in other transactions as well. All these contextual requirements should be handled in a custom make method. One such method for creating quotation from customer dashboard already exists.

Reply:
Passing the data contextually will only solve this use case, but any customizations that depend on the `party_name` trigger will still fail.